### PR TITLE
About section

### PR DESCRIPTION
**Pull request by afmicc**
## About section

Trello: [As a user I should be able to check the about section so I can learn more about Target MVD](https://trello.com/c/c0a8LTUk)

#### Description

* Create `Content` for general purpose to store basic information
* Modify retrieve view to search the content by the key field
* Customize content admin form to display a text area instead of a text field to enter the body value

### CODE REVIEW

File: applications/contents/models.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from django.db import models
+
+
+class Content(models.Model):
+    key = models.CharField(max_length=20, default='', unique=True)
+    title = models.CharField(max_length=100)
+    body = models.CharField(max_length=1000)
```
Comment (brunomichetti) [inline applications/contents/models.py]: If this allows a lot of text then you should use TextField
https://docs.djangoproject.com/en/3.1/ref/models/fields/#django.db.models.TextField

File: applications/contents/serializers.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+from rest_framework import serializers
+
+from .models import Content
+
+
+class ContentSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Content
+        fields = ('key', 'title', 'body', )
```
Comment (brunomichetti) [inline applications/contents/serializers.py]: If you use all the fields you can simply write '__all__'

File: applications/contents/tests/test_retrieve_content.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
+
+    def test_created_content_return_founded(self):
+        content = ContentFactory()
+        self.assertTrue(Content.objects.exists())
+
+        response = self.call_get_content_by_key(content.key)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+        self.user = UserFactory(confirmed=True)
```
Comment (brunomichetti) [inline applications/contents/tests/test_retrieve_content.py:53]: If you are using a user in the most of the tests, maybe you can create it in a setUp or setUpClass function

**Review by brunomichetti — APPROVED** [review]

File: applications/contents/admin.py [hunk 4]
```diff
@@ -0,0 +1,11 @@
+from django.contrib import admin
+
+from .models import Content
+
+
+class ContentAdmin(admin.ModelAdmin):
+    list_display = ('key', 'title', )
+    search_fields = ('key', 'title', )
```
Comment (guilleijo) [inline applications/contents/admin.py]: The trailing comma is used when there is only one field specified to indicate that is a tuple.
```suggestion
    list_display = ('key', 'title')
    search_fields = ('key', 'title')
```

**Review by guilleijo — APPROVED** [review]

**Pull request merged**